views/pools/index.tsx

```diff
@@ -20,7 +20,7 @@ const Pools: FC = () => (
       width={['100%', '100%', '100%', '85%']}
     >
       <Box bg="lowestContainer" borderRadius="2rem" p="xl">
-        Deposit Cards
+        <PoolCard />
```

Comment:
this component must not be inside the "Deposit card", leave it out as an example. because that way it gets polluted and we can't test it uniquely.